### PR TITLE
Upgrade FastEmbed Version

### PR DESCRIPTION
**Pull request by NirantK**
### All Submissions:

* [ ] Contributions should target the `dev` branch. Did you create your branch from `dev`?
* [ ] Have you followed the guidelines in our Contributing document?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [ ] Does your submission pass tests?
2. [ ] Have you installed `pre-commit` with `pip3 install pre-commit` and set up hooks with `pre-commit install`?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: qdrant_client/qdrant_fastembed.py [hunk 1]
```diff
@@ -8,24 +8,22 @@
...
-    "BAAI/bge-small-en": (384, models.Distance.COSINE),
     "BAAI/bge-small-en-v1.5": (384, models.Distance.COSINE),
     "BAAI/bge-base-en-v1.5": (768, models.Distance.COSINE),
     "intfloat/multilingual-e5-large": (1024, models.Distance.COSINE),
 }
 
 
 class QdrantFastembedMixin(QdrantBase):
-    DEFAULT_EMBEDDING_MODEL = "BAAI/bge-small-en"
+    DEFAULT_EMBEDDING_MODEL = "BAAI/bge-small-en-v1.5"
```
[the remark this answers is not in the document]
Comment (joein) [reply qdrant_client/qdrant_fastembed.py:24]: We need to make at least one major release which warns that we are going to change the model

File: qdrant_client/qdrant_fastembed.py [hunk 2]
```diff
@@ -8,24 +8,22 @@
...
 
 try:
-    from fastembed.embedding import DefaultEmbedding
+    from fastembed import TextEmbedding
 except ImportError:
-    DefaultEmbedding = None
+    TextEmbedding = None
 
 SUPPORTED_EMBEDDING_MODELS: Dict[str, Tuple[int, models.Distance]] = {
-    "BAAI/bge-base-en": (768, models.Distance.COSINE),
```
Comment (joein) [inline qdrant_client/qdrant_fastembed.py:16]: we can't just silently remove models which could have already been used by users

**Review by joein — CHANGES_REQUESTED** [review]

**Comment by joein** [comment]
We need to fix our tests to run them without fastembed installed as well

**Comment by joein** [comment]
#522

**Review by generall — APPROVED** [review]

**Review by joein — APPROVED** [review]

**Pull request merged**